apps/frontend/src/app/5_pages/StakePage/components/PersonalStakingStatistics/PersonalStakingStatistics.tsx

```diff
@@ -0,0 +1,118 @@
+import React, { useMemo } from 'react';
+
+import { t } from 'i18next';
+
+import { SupportedTokens } from '@sovryn/contracts';
+import { Heading, HelperButton } from '@sovryn/ui';
+
+import { AmountRenderer } from '../../../../2_molecules/AmountRenderer/AmountRenderer';
+import { TOKEN_RENDER_PRECISION } from '../../../../../constants/currencies';
+import { useAccount } from '../../../../../hooks/useAccount';
+import { translations } from '../../../../../locales/i18n';
+import { decimalic, fromWei } from '../../../../../utils/math';
+import { VP } from '../../StakePage.constants';
+import { PersonalStatistics } from '../../StakePage.utils';
+import { useGetStakingBalanceOf } from '../../hooks/useGetStakingBalanceOf';
+import { useGetStakingStatistics } from '../StakingStatistics/hooks/useGetStakingStatistics';
+import { useGetPersonalStakingStatistics } from './hooks/useGetPersonalStakingStatistics';
+
+export const PersonalStakingStatistics = () => {
```

Comment:
When I sum up individual stakes voting power, it's a different number than `Voting power` in statistics (wallet address `0xaaa5a190ACCbc50F4F9c130b5876521E4D5F9D6c`).

<img width="1495" alt="image" src="https://github.com/DistributedCollective/sovryn-dapp/assets/14878713/26f644e1-7846-4eb5-8e25-d444aa5594fb">